content/events/2020-london/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: London
+date: '2020-09-03'
+end: '2020-09-04'
+location:
+  country: United Kindon
+  city: London
+  url: https://goo.gl/maps/UMW7mDDFvWdvPfXG8
+  venue: Emmanuel Centre
+map: london-map.png
+sched_slug: kubernetescommunitydayslond2020
+social:
+  twitter: 'https://twitter.com/kubedayslondon'
+organizers:
+- name: Cheryl Hung
+  image: cheryl.png
+  twitter: 'https://twitter.com/oicheryl'
+  homepage: 'https://www.oicheryl.com/'
+- name: Dominique Top
+  image: domi.png
+  twitter: https://twitter.com/devopsdom
+- name: Josh Michielsen
+  image: josh.png
+  twitter: https://twitter.com/jmickey_
+  homepage: https://mickey.dev
+- name: Eunice Hung
+  image: eunice.png
+  twitter: https://twitter.com/eunice_serina
+- name: Chris Kranz
+  image: chris.png
+  twitter: https://twitter.com/ckranz
+- name: Gianluca Arbezzano
+  image: gianluca.png
+  twitter: https://twitter.com/GianArb
+  homepage: https://gianarb.it/
+- name: Mihaela Constantinescu
+  image: mihaela.png
+- name: Suraj Narwade
+  image: suraj.png
+  twitter: https://twitter.com/red_suraj
+  homepage: https://suraj.pro
+organizers_email: organisers@kubernetesdays.london
```

Comment:
Hey folks, 

Can you use the KCD provided email address organizers-london@kubernetescommunitydays.org here? It's important that we have year over year context for potential organizers, sponsors, and attendees. It also makes it easier for sponsors who want to sponsor multiple events to "guess" the correct addresses to use as addresses follow the same format across events. 

We are looking into also providing a organisers-london@kubernetescommunitydays.org alias for you. 

Thanks!